### PR TITLE
Update global atmos vminmon job with COMIN/COMOUT for COM prefix

### PR DESCRIPTION
**Pull request by mingshichen-noaa**
# Description
 
NCO has requested that each COM variable specify whether it is an input or an output. This completes that process for the global minimization monitor job.

Refs https://github.com/NOAA-EMC/global-workflow/issues/2451

# Type of change

- Maintenance (code refactor, clean-up, new CI test, etc.)

# Change characteristics
- Is this a breaking change (a change in existing functionality)? NO
- Does this change require a documentation update? NO

# How has this been tested?
 
- Clone and build on RDHPCS
- Cycled tests on Hercules
- Forecast-only tests on Hercules
 
# Checklist
- [x] Any dependent changes have been merged and published
- [x] My code follows the style guidelines of this project
- [x] I have performed a self-review of my own code
- [x] I have commented my code, particularly in hard-to-understand areas
- [x] My changes generate no new warnings
- [x] New and existing tests pass with my changes
- [x] I have made corresponding changes to the documentation if necessary

### CODE REVIEW

File: jobs/JGLOBAL_ATMOS_VMINMON [hunk 1]
```diff
@@ -17,16 +17,18 @@ export gcyc=${GDATE:8:2}
 #############################################
 # TANKverf - WHERE OUTPUT DATA WILL RESIDE
 #############################################
-YMD=${PDY} HH=${cyc} declare_from_tmpl -rx COM_ATMOS_ANALYSIS
-YMD=${PDY} HH=${cyc} declare_from_tmpl -rx COM_ATMOS_MINMON
-YMD=${gPDY} HH=${gcyc} declare_from_tmpl -rx COM_ATMOS_MINMON_PREV:COM_ATMOS_MINMON_TMPL
+YMD=${PDY} HH=${cyc} declare_from_tmpl -rx \
+	COMIN_ATMOS_ANALYSIS:COM_ATMOS_ANALYSIS_TMPL \
+	COMOUT_ATMOS_MINMON:COM_ATMOS_MINMON_TMPL
```
Comment (WalterKolczynski-NOAA) [inline jobs/JGLOBAL_ATMOS_VMINMON]:
```suggestion
	COMOUT_ATMOS_MINMON:COM_ATMOS_MINMON_TMPL
```

**Review by WalterKolczynski-NOAA — CHANGES_REQUESTED** [review]

File: jobs/JGLOBAL_ATMOS_VMINMON (hunk 1 above)
Comment (mingshichen-noaa) [reply jobs/JGLOBAL_ATMOS_VMINMON]: @WalterKolczynski-NOAA  I need to make clear if the following modifications are correct:
1) original lines 20-21 were changed into new lines 20-22
2) original line 22 was changed into new lines 24-25
24 YMD=${gPDY} HH=${gcyc} declare_from_tmpl -rx \ 
25         COMIN_ATMOS_MINMON_PREV:COM_ATMOS_MINMON_TMPL

Comment (WalterKolczynski-NOAA) [reply jobs/JGLOBAL_ATMOS_VMINMON]: The changes are correct. You just need to delete this trailing space.

**Review by KateFriedman-NOAA — APPROVED** [review]
Looks good, thanks @mingshichen-noaa !

**Review by WalterKolczynski-NOAA — APPROVED** [review]

**Pull request merged**